lib/rawscsi/request_signature.rb

```diff
@@ -1,9 +1,8 @@
 module Rawscsi
   class RequestSignature
-    def initialize(*args)
-      options = args.extract_options!
```

Comment:
@ivan-kolmychek `extract_options` is a method in the `activesupport` gem.  But that isn't an official dependency of this gem.  For example, I have a sinatra app that uses rawscsi, that doesn't have activesupport.

Looking here, https://github.com/StevenJL/rawscsi/blob/master/lib/rawscsi/index.rb#L55L69 it looks like the `RequestSignature` initializes with a hash, so I'm just binding the hash to `options`.  Let me know if this looks right.